cmd/scollector/collectors/google_analytics.go

```diff
@@ -58,15 +59,19 @@ func c_google_analytics(clientid string, secret string, tokenstr string, sites [
 		return nil, err
 	}
 
-	dimensions := []string{"browser", "trafficType", "source", "deviceCategory", "operatingSystem"}
+	// dimension: max records we want to fetch
+	// "source" has a very long tail so we limit it to something sane
+	// TODO: Dimensions we want and associated attributes should eventually be
+	// setup in configuration.
+	dimensions := map[string]int{"browser": 0, "trafficType": 0, "source": 10, "deviceCategory": 0, "operatingSystem": 0}
```

Comment:
Named 'top' since I'm sorting and selecting the top X values. Recommendations on a better descriptive name?